Dockerfile

```diff
@@ -9,7 +9,18 @@ RUN CGO_ENABLED=0 GOOS=linux go build -a -o main .
 
 FROM alpine:3.11.3
 
+RUN apk add --update supervisor openvpn curl && rm  -rf /tmp/* /var/cache/apk/*
```

Comment:
This is a very cool idea and setup!
Can you move it to a separate Dockerfile or separate build target in current Dockerfile so that I could make a separate package for it and update instructions? Ideally I want people to have as much options as possible

Reply:
I'll merge and try to achieve that closer to the evening myself if you don't get a chance to do this simply since the idea is very good